### PR TITLE
Create profile

### PR DESCRIPTION
**Pull request by ahal**
This moves profile creation and pref setting to the Raptor constructor. It also removes the 'set_browser_preferences' method, though if desired it's still possible to do this with:
raptor.profile.set_preferences(...)

This also sets up profile creation to get ready for Chrome.

### CODE REVIEW

File: raptor/raptor.py [hunk 1]
```diff
@@ -6,43 +6,48 @@
 
 from __future__ import absolute_import
 
+import json
 import os
 import sys
 
 from mozlog import commandline, get_default_logger
+from mozprofile import create_profile
```
Comment (rwood-moz) [inline raptor/raptor.py:14]: Where is the source for create_profile? I can't seem to find it in:

https://searchfox.org/mozilla-central/source/testing/mozbase/mozprofile/mozprofile

Or is it inherited somewhere?

Comment (rwood-moz) [reply raptor/raptor.py:14]: Oh wait NVM it's in your vendor'ed custom mozbase below haha ok cool

**Comment by rwood-moz** [comment]
Cool I like this, thanks Andrew

File: raptor/raptor.py (hunk 1 above)
Comment (ahal) [reply raptor/raptor.py:14]: Yeah, no worries :). I have mirror patches in mozilla-central clone for all the mozbase changes, so once we go to vendor it'll be easier to get the mozbase changes landed first.

**Pull request merged**